pkgs/applications/networking/browsers/chromium/common.nix

```
@@ -522,6 +522,13 @@ let
           excludes = [ "base/allocator/partition_allocator/src/partition_alloc/*" ];
           revert = true;
         })
+      ]
+      ++ lib.optionals (chromiumVersionAtLeast "131" && stdenv.hostPlatform.isAarch64) [
+        # Reverts decommit pooled pages which causes random crashes of tabs on systems
+        # with page sizes different than 4k. It 'supports' runtime page sizes, but has
+        # a hardcode for aarch64 systems.
+        # https://issues.chromium.org/issues/378017037
+        ./patches/chromium-131-aarch64-page-sizes.patch
```

Instead of vendoring the revert manually we generally prefer referencing original commit with the `revert` flag set. Unfortunately fetching a patch from gitiles is a bit involved but this should work:

```suggestion
        (fetchpatch {
          name = "reverted-v8-decommit-pooled-paged-by-default.patch";
          # https://chromium-review.googlesource.com/c/v8/v8/+/5864909
          url = "https://chromium.googlesource.com/v8/v8/+/1ab1a14ad97394d384d8dc6de51bb229625e66d6^!?format=TEXT";
          decode = "base64 -d";
          stripLen = 1;
          extraPrefix = "v8/";
          revert = true;
          hash = "sha256-PuinMLhJ2W4KPXI5K0ujw85ENTB1wG7Hv785SZ55xnY=";
        })
```

---

I was trying hard to get it to work but couldn't for some reason, I think I was trying
```
extraPrefix = "v8"
```
without the trailing `/`, maybe that was the case :+1:

---

seems to work without trailing `/` too, not sure what I was doing wrong then

---

Without a trailing hash the resulting patch will have `a/v8src/flags/flag-definitions.h` as the path.

Maybe while checking again you ran into the FOD not invalidating, the correct hash for the patch without the trailing slash would be `sha256-HuvfK3biNecn4BGxfKOmfko995wHhYUOQVqfvYB8xEc=`